todo_app/app.py

```diff
@@ -1,14 +1,23 @@
-from flask import Flask
-
+from flask import Flask, render_template, send_from_directory, request
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items  import get_items, add_item
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
 @app.route('/')
 def index():
-    return 'Hello World!'
+    toDoList = get_items()
```

Comment:
Most names should be `snake_case` in Python rather than `CamelCase`. The major exception is class names. See https://www.python.org/dev/peps/pep-0008/ for a detailed style guide.